src/vogsphere.html

```diff
@@ -15,15 +15,33 @@ <h3>Basics</h3>
             <input id="vogsphere__character-name" name="characterName" type="text" required>
         </div>
 
-        <div class="__input-group">
-            <label for="vogsphere__member-group">Member Group</label>
-            <select id="vogsphere__member-group" name="memberGroup" required>
-                <option value="">--Please choose an option--</option>
-                <option value="crew">Crew</option>
-                <option value="scientist">Scientist</option>
-                <option value="passenger">Passenger</option>
-            </select>
-        </div>
+        <!--Added radio buttons for member-group selection-->
+        <fieldset>
+            <legend>
+                Member Group
+            </legend>
+            <!--Selected class minimal to continue same format below-->
+            <ul class="--minimal">
+                <li>
+                    <label for="vogsphere__member-group">
+                        <input type="radio" name="memberGroup" value="crew" id="crew">
```

Comment:
The `id` has to match what's in the corresponding `for`. Without these matching up, screen readers may announce the wrong thing and also clicking on the label to select the radio button doesn't work. See here for more info: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label